csp/adapters/perspective.py

```diff
@@ -13,21 +14,25 @@
 except ImportError:
     raise ImportError("perspective adapter requires tornado package")
 
-
 try:
-    from perspective import PerspectiveManager, Table as Table_, View as View_, __version__, set_threadpool_size
+    from perspective import Server, Table as Table_, View as View_, __version__, set_threadpool_size
```

Comment:
Note this line requires https://github.com/finos/perspective/pull/2831

Reply:
These changes require perspective >= 3.1.4. Not sure how we want to indicate that to users. It's not just the imports, but also the re-introduction of `to_dataframe`